parquet_file/src/metadata.rs

```diff
@@ -228,6 +228,12 @@ pub enum Error {
         expected
     ))]
     IoxMetadataVersionMismatch { actual: u32, expected: Vec<u32> },
+
+    #[snafu(display("Cannot encode ZSTD message: {}", source))]
```

Comment:
It might make sense to note in this error that the problem was encoding parquet metadata (as the use of zstd is likely an implementation detail that could be confusing)

```suggestion
    #[snafu(display("Cannot encode parquet metadata: {}", source))]
```

The same comment applies for the decoding failure.

This is likely not a big deal as either of these errors doesn't seem recoverable so there isn't much a user is going to be able to do about them

Reply:
fixed